src/concurrent/OneToOneRingBuffer.h

```diff
@@ -0,0 +1,236 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <cstddef>
+
+#include "src/concurrent/atomic64.h"
+
+namespace reactivesocket {
+
+namespace RingBufferDescriptor {
+
+constexpr static const size_t CACHE_LINE_LENGTH = 64;
```

Comment:
following fb coding guidelines.. (if you feel we shouldn't follow them, please ignore)
constants should start with k... prefix: kCACHE_LINE_LENGTH